src/database/database.ts

```diff
@@ -266,9 +294,9 @@ export class BotDatabase {
                      .catch((err) => { this.log.error('Could not delete join record!'); });
     }
 
-    public delGuildSettings(guild: Guild): void {
-        this.database.run('DELETE FROM Settings WHERE GuildID = ?', guild.id)
-                     .catch((err) => { this.log.error('Could not delete guild settings!'); });
+    public delTagResponse(name: string, guild: Guild) {

```

Comment:
maybe we should also add delTagResponses to execute when a guild stops using jim, we can get rid of their records